curvefs/src/client/s3/disk_cache_manager_impl.cpp

```diff
@@ -48,10 +48,31 @@ int DiskCacheManagerImpl::Init(const S3ClientAdaptorOption option) {
     }
 
     forceFlush_ = option.diskCacheOpt.forceFlush;
+    threads_ = option.diskCacheOpt.threads;
+    taskPool_.Start(threads_);
     LOG(INFO) << "DiskCacheManagerImpl init end.";
     return 0;
 }
 
+void DiskCacheManagerImpl::Enqueue(
+  std::shared_ptr<PutObjectAsyncContext> context) {
+    auto task = [this, context]() {
+        this->WriteClosure(context);
+    };
+    taskPool_.Enqueue(task);
+}
+
+int DiskCacheManagerImpl::WriteClosure(
+  std::shared_ptr<PutObjectAsyncContext> context) {
+    VLOG(9) << "WriteClosure start, name: " << context->key;
+    WriteDiskCacheClosure done(context);
```

Comment:
Closure is meaningless, it is better to call directly

Reply:
fix